setup.py

```diff
@@ -1,26 +1,38 @@
 from setuptools import setup, find_packages
+from subprocess import check_output
+
+
+def get_version():
+    # https://github.com/uc-cdis/dictionaryutils/pull/37#discussion_r257898408
+    try:
+        tag = check_output(
+            ["git", "describe", "--tags", "--abbrev=0", "--match=[0-9]*"]
+        )
+        return tag.strip("\n")
+    except Exception:
+        # if somehow you get the repo not from git,
+        # hardcode default major version
+        return "1.0.0"
```

Comment:
TBH I don't think we should fall back to a default version, because that may cause troubles and doesn't really solve the problem here.

Reply:
so what do we do as an alternative?

Reply:
I think we should let the error propagate to the user with a user-friendly message, saying something like "The version number cannot be extracted from git tag in this source distribution; please either download the source from PyPI, or check out from GitHub and make sure that git is available."

Reply:
seems fine 👍

Reply:
@philloooo thanks for checking up! If this PR's good, I'll propagate to all PyPI-releasing repositories + template?